lib/modules/versioning/types.ts

```diff
@@ -33,6 +33,15 @@ export interface VersioningApi {
   matches(version: string, range: string | Range): boolean;
 
   valueToVersion?(version: string): string;
+
+  /**
+   * @returns true if sub range is entirely contained by dom range, false otherwise,
+   * and undefined if it cannot determine it.
+   *
+   * @param sub - the sub range
+   * @param dom - the dom range
+   */
+  subset?(sub: string, dom: string): boolean | undefined;
```

Comment:
```suggestion
  subset?(subRange: string, superRange: string): boolean | undefined;
```
can we use the semver parameter names and description, seems more easy to understand

https://www.npmjs.com/package/semver